src/sdk/namespace_fs.js

```diff
@@ -79,6 +79,46 @@ function sort_entries_by_name(a, b) {
     return 0;
 }
 
+function _get_mtime_from_filename(filename) {
+    if (!filename.includes('_mtime-')) {
+        // Latest file wont have time suffix which will push the latest
+        // object last in the list. So to keep the order maintained,
+        // returning the latest time. Multiplying with 1e6 to provide
+        // nano second precision
+        return Date.now() * 1e6;
+    }
+    const file_parts = filename.split('-');
+    return parseInt(file_parts[1], 36);
```

Comment:
once you get the mtime part, you should use `size_utils.string_to_bigint` and not `parseInt`
e.g.: see `_extract_version_info_from_xattr`

Reply:
Checking for BigInt supported operations